app/assets/javascripts/modules/order/views/history_item.coffee

```diff
@@ -0,0 +1,19 @@
+@Lunchiatto.module 'Order', (Order, App, Backbone, Marionette, $, _) ->
+  Order.HistoryItem = Marionette.ItemView.extend
+    template: 'orders/history_item'
+    tagName: 'li'
+    className: 'hover-pointer'
+
+    triggers:
+      'click': 'show:order'
+
+    behaviors:
+      Animateable:
+        types: ['fadeIn']
+
+    templateHelpers: ->
```

Comment:
Where do we use those helpers?

Reply:
They are responsible for display of status (i.e. "delivered" in green)